.github/CODEOWNERS

```diff
@@ -8,19 +8,80 @@
 # The go-samples-reviewers team is the default owner for anything not
 # explicitly taken by someone else.
 *                                     @GoogleCloudPlatform/go-samples-reviewers
+/docs/                                @GoogleCloudPlatform/go-samples-reviewers
+/getting-started/                     @GoogleCloudPlatform/go-samples-reviewers
+/internal/                            @GoogleCloudPlatform/go-samples-reviewers
+/testing/                             @GoogleCloudPlatform/go-samples-reviewers
 
+# CNDB, Storage, and Infra-DB
 /bigtable/                            @GoogleCloudPlatform/go-samples-reviewers @GoogleCloudPlatform/cloud-native-db-dpes
 /cloudsql/                            @GoogleCloudPlatform/go-samples-reviewers @GoogleCloudPlatform/infra-db-dpes
 /datastore/                           @GoogleCloudPlatform/go-samples-reviewers @GoogleCloudPlatform/cloud-native-db-dpes
 /firestore/                           @GoogleCloudPlatform/go-samples-reviewers @GoogleCloudPlatform/cloud-native-db-dpes
+/spanner/                             @GoogleCloudPlatform/go-samples-reviewers @GoogleCloudPlatform/infra-db-dpes
```

Comment:
Could we make these infra-db-sdk while you're here? We renamed the team.

Reply:
And technically, spanner should be under the "does not have an owner" category. We never touch Spanner stuff.